docs/parseparam.rst

```diff
@@ -0,0 +1,43 @@
+#This document includes the details on the all the parse-able params in Quokka.
```

Comment:
This does not need to be a comment. It can just be plain text, e.g.: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/topics.html

Reply:
Does it look better now?

Reply:
You can preview how it looks by clicking on "Files changed" -> "..." -> "View file": https://github.com/quokka-astro/quokka/blob/2ce3caca1453d3cf7306c8b228bc26da65091ab2/docs/parseparam.rst

Reply:
Something is not quite right with the formatting.